index.d.ts

```diff
@@ -0,0 +1,2148 @@
+  /**
+   * Current status. 
+   * Possible values are "creating", "created" and "failed"
+   */
+  status: "creating" | "created" | "failed";
+
+  /**
+   * Human readable message explaining any failures
+   */
+  message: string;
+
+  /**
+   * Address that the Shipments will be shipped from
+   */
+  address: IAddress;
+
+  /**
+   * Tracking codes included on the ScanForm
+   */
+  tracking_codes: string[];
+
+  /**
+   * Url of the document
+   */
+  form_url: string;
+
+  /**
+   * File format of the document
+   */
+  form_file_type: string;
+
+  /**
+   * The id of the associated Batch. Unique, starts with "batch_"
+   */
+  batch_id: string;
+}
+
+/**
+ * @todo PostageLabel hasn't be documented - https://www.easypost.com/docs/api/node
+ */
+export declare interface IPostageLabel extends IObjectWithId<"PostageLabel">, IDatedObject {
+  date_advance: number;
+  integrated_form: string;
+  label_date: string;
+  label_resolution: number;
+  label_size: string;
+  label_type: string;
+  label_url: string;
+  label_file_type: string;
+  label_pdf_url: string;
+  label_epl2_url: string;
+  label_zpl_url: string;
+}
+
+
+/**
+ * Each Webhook contains the url which EasyPost will notify whenever an object in our system updates. 
+ * Several types of objects are processed asynchronously in the EasyPost system, so whenever an object updates, an Event is sent via HTTP POST to each configured webhook URL. 
+ * The Webhook object provides CRUD operations for all Webhooks.
+ * 
+ * Currently, our recommended best practice for securing Webhooks involves using basic authentication and HTTPS on your endpoint. 
+ * This will help prevent any altering of any information communicated to you by EasyPost, prevent any third parties from seeing your webhooks in transit, and will prevent any third parties from masquerading as EasyPost and sending fraudulent data. 
+ * EasyPost performs certificate validation and requires any TLS-enabled (HTTPS) webhook recipients to have a certificate signed by a public trusted certification authority. 
+ * We do not support sending webhooks to over SSLv2, SSLv3, or any connection using so-called export-grade ciphers. 
+ * For documentation on how to set up your server with TLS, we recommend Mozilla's guide to Server-Side TLS and Qualys's SSL/TLS deployment best practices guide.
+ * 
+ * In general, a Webhook's endpoint should return a status code of 2XX. 
+ * A 200 is preferred, but any 2XX status will indicate to our system that the Webhook request was successful. 
+ * Endpoints that return a large volume and rate of failures over a period of time will get automatically disabled by the system; a disabled Webhook can be re-enabled using the Webhook update endpoint.
+ * 
+ * @todo [Webhook Object] anchor links to Report Object - need to fix
+ * @see https://www.easypost.com/docs/api/node#webhooks
+ */
+export declare interface IWebhook extends IObjectWithId<"Webhook"> {
```

Comment:
The [Webhook](https://www.easypost.com/docs/api/node#webhooks) Object anchor links to the Report Object instead.

![image](https://user-images.githubusercontent.com/2395597/104793048-cb137080-576e-11eb-930d-f23a162958c0.png)

Reply:
Heh, nice find! I'll put up a fix for this right now internally.